### PR TITLE
Stack of Vampyre fixes

### PR DESCRIPTION
**Pull request by jaspercb**
* [vampyre] Buy Torso Awaregness if we have the Garbage Tote
* [vampyre] Use forms that turn off Baleful Howl when we don't need a banish but could benefit from +item or whatever
* [vampyre] Pull gauze garter before checking for diet pill use

### CODE REVIEW

File: RELEASE/scripts/sl_ascend/sl_picky.ash [hunk 1]
```diff
@@ -74,6 +74,16 @@ void picky_startAscension()
 
 boolean picky_buyskills()
 {
+	// TODO: Torso Awaregness is worth obtaining in other cases too.
+	if(my_meat() >= 10000
+	   && (my_class() == $class[Vampyre])
+	   && gnomads_available()
+	   && (!have_skill($skill[Torso Awaregness]))
+	   && (item_amount($item[January\'s Garbage Tote]) != 0)
+	   && (!is_unrestricted($item[January\'s Garbage Tote])))
```
Comment (soolar) [inline RELEASE/scripts/sl_ascend/sl_picky.ash]: why `!is_unrestricted` instead of `is_unrestricted`?

Comment (jaspercb) [reply RELEASE/scripts/sl_ascend/sl_picky.ash]: Nice catch! It's totally wrong and is fixed in a later commit in the stack.

Comment (soolar) [reply RELEASE/scripts/sl_ascend/sl_picky.ash]: never mind you fixed that in a later commit

**Comment by jaspercb** [comment]
This resolves #113

**Pull request merged**